lib/src/flex/types.ts

```diff
@@ -0,0 +1,41 @@
+type Props = {
+  direction?: "row" | "row-reverse" | "column" | "column-reverse";
+  wrap?: "nowrap" | "wrap" | "wrap-reverse";
+  justifyContent?:
+    | "flex-start"
+    | "flex-end"
+    | "start"
+    | "end"
+    | "left"
+    | "right"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "space-evenly";
+  alignItems?:
+    | "stretch"
+    | "flex-start"
+    | "flex-end"
+    | "start"
+    | "end"
+    | "self-start"
+    | "self-end"
+    | "center"
+    | "baseline";
+  alignContent?:
+    | "normal"
+    | "flex-start"
+    | "flex-end"
+    | "start"
+    | "end"
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "space-evenly"
+    | "stretch";
+  gap?: string;
```

Comment:
I think the gap's type should be: `string | Gap`, where the Gap type is `{ rowGap: string, columnGap: string }`. What do you think? I know that with the current type string the user can write "20px 10px" and would work fine but I think the new type could orientate him/her better.

Reply:
Yes, it is a good solution ✌🏼